var/spack/repos/builtin/packages/procps/libintl.patch

```diff
@@ -0,0 +1,10 @@
+--- a/Makefile.am	2023-06-29 14:12:25.671539354 -0500
++++ b/Makefile.am	2023-06-29 14:26:58.852425405 -0500
+@@ -1,6 +1,5 @@
+-CYGWINFLAGS =
++CYGWINFLAGS = $(LTLIBINTL)
```

Comment:
@greenc-FNAL 

What sets this? IIRC `libintl` may not be present, so presumably this is useful when the variable is unset as well (which indicates that no additional `-lintl` flag is needed).

Reply:
`gettext.m4`

Reply:
Thanks, I see it's convenient to set CYGWINFLAGS because it is used for defining LDFLAGS for many of the objects (regardless of whether we are building on Cygwin)